.ci/pytorch/cpp_doc_push_script.sh

```diff
@@ -69,6 +69,13 @@ pushd cppdocs
 mkdir /tmp/cppdocs-sync
 mv _config.yml README.md /tmp/cppdocs-sync/
 rm -rf ./*
+rm -rf _static/*
+# Verify _static is empty
+if [ -n "$(ls -A _static 2>/dev/null)" ]; then
+  echo "Warning: _static directory is not empty after cleanup"
+  ls -la _static
+fi
+
```

Comment:
Why this is needed? `rm -rf` on the previous line should be enough?

Reply:
cleaned up